tests/generator.js

```diff
@@ -6,11 +6,21 @@ const salt = bcrypt.genSaltSync(10);
 
 function generateUser(password) {
 	const email = faker.internet.email();
+
 	return {
 		userName: faker.internet.userName().toLowerCase(),
 		displayName: faker.name.findName(),
 		email: email.toLowerCase(),
 		displayEmail: email,
+		location: faker.fake('{{address.cityPrefix}} {{name.firstName}}{{address.citySuffix}}, {{address.countryCode}}'),
+		dateOfBirth: faker.date.past(50, new Date()).toISOString(),
```

Comment:
Not urgent but this can result in some toddler divers! lol Perhaps subtract ~12 years from this? Mabe use `moment`?